libraries/AP_LandingGear/AP_LandingGear.cpp

```diff
@@ -32,13 +35,55 @@ const AP_Param::GroupInfo AP_LandingGear::var_info[] = {
     // @Values: 0:WaitForPilotInput, 1:Retract, 2:Deploy
     // @User: Standard
     AP_GROUPINFO("STARTUP", 2, AP_LandingGear, _startup_behaviour, (uint8_t)AP_LandingGear::LandingGear_Startup_WaitForPilotInput),
+    
+    // @Param: DEPLOY_PIN
+    // @DisplayName: Chassis deployment feedback pin
+    // @Description: Pin number to use for feedback of gear deployment. If set to -1 feedback is disabled.
+    // @Values: -1:Disabled,50:PX4 AUX1,51:PX4 AUX2,52:PX4 AUX3,53:PX4 AUX4,54:PX4 AUX5,55:PX4 AUX6
+    // @User: Standard
+    // @RebootRequired: True
+    AP_GROUPINFO("DEPLOY_PIN", 3, AP_LandingGear, _pin_deployed, -1),
+
+    // @Param: DEPLOY_POL
+    // @DisplayName: Chassis deployment feedback pin polarity
+    // @Description: Polarity for feedback pin. If this is 1 then the pin should be high when gear are deployed. If set to 0 then then deployed gear level is low.
+    // @Values: 0:Low,1:High
+    // @User: Standard
+    AP_GROUPINFO("DEPLOY_POL", 4, AP_LandingGear, _pin_deployed_polarity, 0),
+    
+    // @Param: WOW_PIN
+    // @DisplayName: Weight on wheels feedback pin
+    // @Description: Pin number to use for feedback of weight on wheels condition. If set to -1 feedback is disabled.
+    // @Values: -1:Disabled,50:PX4 AUX1,51:PX4 AUX2,52:PX4 AUX3,53:PX4 AUX4,54:PX4 AUX5,55:PX4 AUX6
+    // @User: Standard
+    // @RebootRequired: True
+    AP_GROUPINFO("WOW_PIN", 5, AP_LandingGear, _pin_weight_on_wheels, DEFAULT_PIN_WOW),
 
+    // @Param: WOW_POL
+    // @DisplayName: Weight on wheels feedback pin polarity
+    // @Description: Polarity for feedback pin. If this is 1 then the pin should be high when there is weight on wheels. If set to 0 then then weight on wheels level is low.
+    // @Values: 0:Low,1:High
+    // @User: Standard
+    AP_GROUPINFO("WOW_POL", 6, AP_LandingGear, _pin_weight_on_wheels_polarity, DEFAULT_PIN_WOW_POL),
+    
     AP_GROUPEND
 };
 
+AP_LandingGear *AP_LandingGear::_instance;
```

Comment:
This should be called singleton, rather then _instance. Instance is to overloaded a meaning in the various drivers.

Reply:
fixed